cob_gazebo_objects/objects/gitterbox.urdf

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" ?> 
+<robot name="my_object" static="true">
+  <link name="base_link">
+  <inertial>
+    <mass value="1.37" />
+    <origin xyz="0 0 0.0" />
+    <inertia ixx="1.0" ixy="0.0" ixz="0.0" iyy="1.0" iyz="0.0" izz="1.0"/>
+  </inertial>
```

Comment:
Could you use proper box_inertial from [here](https://github.com/ipa320/cob_common/blob/indigo_dev/cob_description/urdf/common.xacro#L27-L35)...maybe we should add a comon.xacro within cob_gazebo_objects to reduce dependencies...

BTW, I guess if you use static=true, inertial is obosolete anyway?